packages/account/src/providers/provider.ts

```diff
@@ -598,39 +598,41 @@ export default class Provider {
     await this.fetchChainAndNodeInfo();
   }
 
+  async refreshCacheFunction(): Promise<{ chain: ChainInfo; nodeInfo: NodeInfo }> {
```

Comment:
```suggestion
  private async refreshCacheFunction(): Promise<{ chain: ChainInfo; nodeInfo: NodeInfo }> {
```

But I'm thinking we could just do what's mentioned in @arboleya's [comment](https://github.com/FuelLabs/fuels-ts/pull/3324/files#r1798469752) and call it a day.

Reply:
c3eadcea4